### ISSUE TITLE
Viewing attachment from user->my ticket cause imagejpeg expect 1 param but boolean given

### ISSUE THREAD

**Issue by thanhpt** [ladybirdweb/faveo-helpdesk#143]
I attached an xls file to the ticket and it is viewed ok at agent's ticket timeline but when I opened it at user->check my ticket then imagejpeg appeared.

Why : look at the below code 

```
foreach ($attachments as $attachment) {
            if ($attachment->type == 'pdf') {

            } elseif ($attachment->type == 'docx') {

            } else {
                $image = @imagecreatefromstring($attachment->file);
                ob_start();
                imagejpeg($image, null, 80);
                $data = ob_get_contents();
                ob_end_clean();
                $var = '<img src="data:image/jpg;base64,' . base64_encode($data) . '" />';
                $body = str_replace($attachment->name, "data:image/jpg;base64," . base64_encode($data), $body);

                $string = $body;
                $start = "<head>";
                $end = "</head>";
                if (strpos($string, $start) == false || strpos($string, $start) == false) {

                } else {
                    $ini = strpos($string, $start);
                    $ini += strlen($start);
                    $len = strpos($string, $end, $ini) - $ini;
                    $parsed = substr($string, $ini, $len);
                    $body2 = $parsed;
                    $body = str_replace($body2, " ", $body);
                }
            }
        }
```

Seem that everything which is not pdf or docx will be treated as jpg or png which is not correct.
So the last else should be change to 
 `else if ($attachment->type == 'jpg' || $attachment->type == 'png') {`

as it is used in tickettimle.blade.php

**Comment by mverma16** [comment]
has been fixed and will be a part of next version.

**Comment by azprrnrt** [comment]
Hi
Issue not totally solved.
Patch _Bug-fix-patch4_ (commit 6f7ee1569cdd7ce297a0f6e5df482427ec0ca1a6) fixed it on file:
resources/views/themes/default1/client/helpdesk/ckeckticket.blade.php
This same fix should be apply on file:
resources/views/themes/default1/client/helpdesk/unauth/**showticket.blade.php**

imagejpeg returning exception:

> imagejpeg() expects parameter 1 to be resource, boolean given
